src/Solnet.Programs/StakeProgram.cs

```diff
@@ -0,0 +1,365 @@
+﻿using Solnet.Programs.Models;
+using Solnet.Programs.Utilities;
+using Solnet.Rpc.Models;
+using Solnet.Wallet;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using static Solnet.Programs.Models.Stake.Instruction;
+using static Solnet.Programs.Models.Stake.State;
+
+namespace Solnet.Programs
+{
+    /// <summary>
+    /// Implements the Stake Program methods.
+    /// <remarks>
+    /// For more information see:
+    /// LINK HERE
+    /// LINK HERE
+    /// </remarks>
+    /// </summary>
+    public static class StakeProgram
+    {
+        /// <summary>
+        /// The public key of the Stake Program.
+        /// </summary>
+        public static readonly PublicKey ProgramIdKey = new("Stake11111111111111111111111111111111111111");
+        /// <summary>
+        /// The public key of the Recent Block Hashes System Variable. 
+        /// </summary>
+        public static readonly PublicKey
+            SysVarRecentBlockHashesKey = new("SysvarRecentB1ockHashes11111111111111111111");
+
+        /// <summary>
+        /// The public key of the Rent System Variable.
+        /// </summary>
+        public static readonly PublicKey SysVarRentKey = new("SysvarRent111111111111111111111111111111111");
```

Comment:
I would create a new static class for all of these sysvars

Reply:
Moved SysVars to static class located at Solnet.Programs.SysVars